content/ja/blog/_posts/2023-01-06-unhealthy-pod-eviction-policy-for-pdb.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: blog
+title: "Kubernetes 1.26: PodDisruptionBudgetsによって保護された不健全なPodに対する退避ポリシー"
+date: 2023-01-06
+slug: "unhealthy-pod-eviction-policy-for-pdbs"
+---
+
+**著者:** Filip Křepinský (Red Hat), Morten Torkildsen (Google), Ravi Gudimetla (Apple)
+
+
+アプリケーションの中断がその可用性に影響を与えないようにすることは、簡単な作業ではありません。
+先月リリースされたKubernetes v1.26では、[PodDisruptionBudgets](/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) (PDB) に
+_不健全なPodの退避ポリシー_ を指定して、ノード管理操作中に可用性を維持できるようになりました。
+この記事では、アプリケーション所有者が中断をより柔軟に管理できるようにするために、PDBにどのような変更が導入されたのかを詳しく説明します。
+
+## これはどのような問題を解決しますか？{#what-problem-does-this-solve}
+
+APIによって開始されるPodの退避では、PodDisruptionBudgets(PDB)が考慮されます。
+これは、退避によるPodへの[自発的な中断](/ja/docs/concepts/scheduling-eviction/#pod-disruption)の要求は保護されたアプリケーションを中断してはならず、
+PDBの`.status.currentHealthy`が`.status.desiredHealthy`を下回ってはいけないことを意味します。
+[Unhealthy](/docs/tasks/run-application/configure-pdb/#healthiness-of-a-pod)な実行中のPodはPDBステータスにはカウントされませんが、
+これらの退避はアプリケーションが中断されない場合にのみ可能です。
+これにより、中断されたアプリケーションやまだ開始されていないアプリケーションが、退避によって追加のダウンタイムが発生することなく、できるだけ早く可用性を達成できるようになります。
+
+残念ながら、これは手動の介入なしでノードをドレインしたいクラスター管理者にとって問題を引き起こします。
+(バグまたは構成ミスにより)Podが`CrashLoopBackOff`状態になっているアプリケーション、
+または単に準備ができていないPodがあるアプリケーションが誤動作している場合、このタスクはさらに困難になります。
```

Comment:
```suggestion
(バグまたは構成ミスにより)Podが`CrashLoopBackOff`状態になっているアプリケーション、または単に準備ができていないPodがあるアプリケーションが誤動作している場合、このタスクはさらに困難になります。
```

We can just merge lines if it's a single sentence. Breaking lines will insert a space but it's not expected for Japanese